### PR TITLE
[feature](multi-catalog) support trino jdbc catalog and jdbc external table

### PR DESCRIPTION
**Pull request by zy-kkk**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [x] Has document been added or modified
* [ ] Does it need to update dependencies
* [x] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zy-kkk** [comment]
run buildall

**Comment by zy-kkk** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 35.46 seconds
 stream load tsv:          426 seconds loaded 74807831229 Bytes, about 167 MB/s
 stream load json:         23 seconds loaded 2358488459 Bytes, about 97 MB/s
 stream load orc:          72 seconds loaded 1101869774 Bytes, about 14 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230410033827_clickbench_pr_126718.html

File: docs/en/docs/lakehouse/external-table/jdbc.md [hunk 1]
```diff
@@ -180,9 +180,9 @@ Test information on more versions will be provided in the future.
 | 22           | clickhouse-jdbc-0.3.2-patch11-all.jar |
 | 22           | clickhouse-jdbc-0.4.1-all.jar         |
 
-#### 6.Sap_HanaTest
+#### 6.Sap Hana Test
```
Comment (morningman) [inline docs/en/docs/lakehouse/external-table/jdbc.md:183]: Why using `Test`?

**Review by morningman — COMMENTED** [review]
Do we support insert into trino?

**Comment by zy-kkk** [comment]
> Do we support insert into trino?

yes,support

File: docs/en/docs/lakehouse/external-table/jdbc.md (hunk 1 above)
Comment (zy-kkk) [reply docs/en/docs/lakehouse/external-table/jdbc.md:183]: Here's a list of the versions we tested

**Comment by zy-kkk** [comment]
run buildall

**Comment by zy-kkk** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**